integrations/upgrade/index.test.ts

```diff
@@ -1584,6 +1712,10 @@ test(
 
       @config '../tailwind.config.ts';
 
+      --- ./src/tailwind-setup.utilities.css ---
+      @import './components.utilities.css';
+      @import './utilities.utilities.css'
```

Comment:
Good catch, _a_ split made sense, but the split should've stopped at the Tailwind root file indeed. Handled here: https://github.com/tailwindlabs/tailwindcss/pull/14982/commits/b62c49990dfdc3aa37fb0b6b2afaf8b3e6d6dc41

You can see the difference in the integration tests: https://github.com/tailwindlabs/tailwindcss/pull/14982/commits/87c0bb29e55a939cad795ad318c6300d65c4452e

Re-ran it on the Commit template and the output is the same (because no splits happened there, but it also didn't break anything)